pkg/epp/scheduling/framework/plugins/picker/common.go

```diff
@@ -17,7 +17,7 @@ limitations under the License.
 package picker
 
 const (
-	DefaultMaxNumOfEndpoints = 1 // common default to all pickers
+	DefaultMaxNumOfEndpoints = 2 // common default to all pickers, should have atleast one fallback
```

Comment:
this should be discussed, I would keep out of current PR (personally I would also leave on 1).
adding the mechanism is one thing, deciding on defaults is another.